lib/python/treadmill_aws/cli/admin/cell/configure_partition.py

```diff
@@ -58,9 +58,9 @@ def configure_partition_cmd(autoscale_max,
             data = {}
 
         autoscale = data.get('autoscale', {})
-        if autoscale_max:
+        if autoscale_max is not False:
```

Comment:
why?

Reply:
0 is falsey- it's not possible to set autoscale_min or _max to '0'.

Reply:
"is not None" then?

Reply:
You're right, fixed.